docs/Changelog.md

```diff
@@ -28,6 +28,7 @@
 * [#3709](https://github.com/TouK/nussknacker/pull/3709) Fix BestEffortJsonSchemaEncoder - fix encoding only strings when `"type": String`
 * [#3707](https://github.com/TouK/nussknacker/pull/3707) Handling apikey in OpenAPI
 * [#3716](https://github.com/TouK/nussknacker/pull/3716) Fix: JsonSchema sinks allow to add additional fields also in `strict validation mode`, if schema permits them.
+* [#3727](https://github.com/TouK/nussknacker/pull/3727) Improvements: Change RR Sink validator from JsonSchemaSubclassDeterminer to JsonSchemaOutputValidator
```

Comment:
From the user's perspective, it is a technical detail, maybe we can write some short human-friendly explanation?